client/state/post-formats/reducer.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { isValidStateWithSchema } from 'state/utils';
+import * as schema from './schema';
```

Comment:
It's probably fine here since there's only a single export anyways, but in generally we should avoid `import * as` since it can't be optimized with future [tree shaking in Webpack 2](http://www.2ality.com/2015/12/webpack-tree-shaking.html).